docs/_data/paths/rating-proposals_realtime.yaml

```diff
@@ -1,12 +1,87 @@
+
+    Clients SHOULD perform Conditional `GET` using the `If-None-Match` header
+    and the `ETag` of a previous `GET` response to poll this endpoint. Rate
+    limiting is done on a per Ratings Provider basis, so requests from
+    independent clients used by the same provider count against the same quota.
+
+  tags:
+    - Rating Proposals
+    - Real-Time
+  responses:
+    '200':
+      description: OK
+      content:
+        application/vnd.trolie.rating-realtime-proposal-status.v1+json: &status-response
+          schema:
+            $ref: ../components/schemas/array-max-monitored-elements.yaml#/real-time-proposal-status
+          example:
+            $ref: ../../example-narratives/examples/realtime-proposal-status.json
+      headers:
+        $ref: '../openapi-split.yaml#/components/responses/204/headers'
+
+    '304':
+      $ref: '../openapi-split.yaml#/components/responses/304'
+    '400': &malformed
+      $ref: '../openapi-split.yaml#/components/responses/400-problem'
+    '401': &unauthorized-empty
+      $ref: '../openapi-split.yaml#/components/responses/401-empty'
+    '403': &forbidden-empty
+      $ref: '../openapi-split.yaml#/components/responses/403-empty'
+    '406': &not-acceptable-empty
+      $ref: '../openapi-split.yaml#/components/responses/406-empty'
+    '429': &rate-limit-hit
+      $ref: '../openapi-split.yaml#/components/responses/429-empty'
+    '500': &unexpected-error-empty
+      $ref: '../openapi-split.yaml#/components/responses/500-empty'
+    default: *unexpected-error-empty
+  security:
+    - oauth2-primary-flow:
+        - read:realtime-proposals
 post:
   operationId: postRealTimeProposal
   summary: Submit Real-Time Rating Proposal
   description: |
 
-    The Clearinghouse will make use of the most recent Ratings Proposals it has
-    received when determining Real-Time Limits. Ratings Providers with Real-Time
-    Ratings Obligations should submit their Ratings Proposals using this
-    operation.
+    At the discretion of the TROLIE server owner, Ratings Providers use 
+    this endpoint to submit ratings within the current hour.  
+    These are assumed to be real-time ratings, based on measurements
+    of ambient conditions as opposed to forecasts.  These ratings will be 
+    used by Transmission Providers in real-time grid operations processes, 
+    such as state estimator and real-time markets.  The clearinghouse for real-time ratings
+    may be run more frequently than the one for forecast ratings to adapt to real-world 
+    conditions.  
+
+    These ratings may be either AARs or DLRs.  
+
+    The API mechanics are different than forecasts, as there is no implicitly created
+    time window to modify.  Ratings providers simply `POST` new values as they are measured and computed.  
+
+    Rules for usage however are similar to forecasts; real-time proposals do not have to contain
+    every resource for which the Ratings Provider is responsible.  Data may be broken into batches
```

Comment:
I have some reservations about not fulfilling all the RT obligations in one message, but I'm struggling to articulate them.

I do want to ask about how the status works when the Clearinghouse sends its audit report with ratings coming in from ICCP.

Reply:
Just to capture, we agreed this is something we may want to revisit later.